inc/AMP/Component.php

```diff
@@ -76,7 +77,7 @@ public function action_add_amp_support() {
 	 * @return bool Whether the AMP plugin is active and the current request is for an AMP endpoint.
 	 */
 	public function is_amp() : bool {
-		return function_exists( 'is_amp_endpoint' ) && \is_amp_endpoint();
+		return function_exists( 'is_amp_endpoint' ) && is_amp_endpoint();
```

Comment:
I'd prefer if we didn't import this one because it's not from core and might not exist.

I know importing it without it existing wouldn't cause an actual issue, but I think it's good to make that separation between core functions (which always exist) and plugin functions (which might exist).

Reply:
Ok. Updated following your advice.